### ISSUE TITLE
Mac: Can't open files of this type

### ISSUE THREAD

**Issue by ghost** [nzbgetcom/nzbget#86]
Getting this when double clicking an NZB in Finder on Mac: 
![CleanShot 2024-01-06 at 11 31 06@2x](https://github.com/nzbgetcom/nzbget/assets/8832687/9dda6459-7f7f-48ae-bde4-9b62da63bf5e)

**Comment by dnzbk** [comment]
Yes, we are aware of this problem. I'll take a look at it as soon as I have some free time.

**Comment by dnzbk** [comment]
Uh, sure. 
Sorry, very busy right now, but I'll try to find time and fix this problem soon.

**Comment by dnzbk** [comment]
I have tried to figure out the problem but unfortunately I haven't succeeded so far. 
I have very little experience with Cocoa and Objective-c to fix it right now. 
I'll come back to this problem a little later. 
Sorry for that.
